data/simpy.gram

```diff
@@ -441,12 +442,12 @@ star_atom[expr_ty]:
 
 paren_ann_assign_target[expr_ty]:
```

Comment:
Suggestion: rename this to `inside_paren_ann_assign_target` now that we are at it